contents/handbook/small-teams/product-analytics/objectives.mdx

```diff
@@ -1,12 +1,16 @@
-## Q4 2023 Goals
+## Q1 2024 Goals
 
-1. Project "NoteForce Exploration 3000" (Michael, Thomas)
-  - Pair up with Team Replay to complete projects Data Exploration, Notebooks and PostHog 3000.
-  - Release new navigation and insight editor.
-  - Work more HogQL prompts into the interface.
-  - Build metrics into the system, introduce Metric Mondays.
+1. HogQL & Data Exploration (Julian, Marius)
+  - Convert ALL our insights to use HogQL as their base.
+  - Improve the type system and get rid of `assumeNotNull`.
+  - Write great docs for HogQL and data exploration nodes.
+  - Proactively monitor query performance.
 
-2. Project "Finish the darned Query Engine" (Tom, Marius)
-  - Finish insight HogQL conversion.
-  - Support non-event data sources on HogQL insights (BI tools for data warehouse).
-  - Build tools to measure HogQL query performance, and catch regressions before they hit users.
+2. Product Analytics frontend (Thomas, Michael)
+  - Clean things up now that we have PostHog 3000 and HogQL everywhere.
+  - Work on the universal "Explore" view. 
```

Comment:
I feel like this goal is super loose. The actual session output had some more concrete things like:
- Reworked filtering experience
- Query builder 3000
- Notebook-first flow
- Purge legacy UI
- Make Product Analytics as easy as Web Analytics

Any reason why not to just put them here? Otherwise the goal and the item mentioned (especially the first) feel like non-statements...

Reply:
I explicitly didn't go into specifics, as we don't know what we'll actually end up working on... but added them now as inspiration.